website/content/en/docs/main/tutorials/bookinfo/index.md

```diff
@@ -1,16 +1,16 @@
 ---
 title: BookInfo
-description: Running BookInfo application with different policies.
+description: Running BookInfo application with different policies
 ---
 
 
-The tutorial sets up [Istio BookInfo application][] in different clusters.
+The tutorial sets up the [Istio BookInfo application][] in different clusters.
 The tutorial demonstrates the use of AccessPolicy and PrivilegedAccessPolicy custom resources.
-The tutorial shows different load-balancing policies like: random, round-robin or static destination.
+The tutorial shows different load-balancing policies like: random, round robin or static destination.
 For more details, see the [policies documentation][].
 This test creates three kind clusters:
 
-* Two Product-Page microservice (application frontend) and details microservice run on the first cluster.
+* Two Product Page microservices (application frontend) and a details microservice run on the first cluster.
```

Comment:
I think the microservice name is `Details` and not `details` - same as `Product Page`